components/lib/Head/Head.tsx

```diff
@@ -12,20 +12,21 @@ export const Head: FC<IHeadProps> = (props) => {
     publishedDate,
     description = '',
   } = props;
+  const cleanedTitle = title.replaceAll(/&nbsp;/g, ' ');
```

Comment:
- use unicode of &nbsp; for replacing
- create an utility function (not in utils folder, just in another file)

Reply:
What is the benefit of this movement of code?

Reply:
clean code: you can name the function and another developer can understant what is happening here